projects/kit/components/input-files/input-files.component.ts

```diff
@@ -178,6 +178,7 @@ export class TuiInputFilesComponent
             return;
         }
 
+        this.input?.nativeElement.focus();
```

Comment:
The issue is `(mousedown.prevent)` in the template. I don't know what it was there for, seems to have been there from the beginning. We probably can drop it.

Reply:
fixed